src/plugins/gstreamer/camerabin/camerabinexposure.cpp

```diff
@@ -118,13 +118,13 @@ QVariantList CameraBinExposure::supportedParameterRange(ExposureParameter parame
             g_object_get(G_OBJECT(m_session->cameraSource()), "supported-iso-speeds", &iso_modes, NULL);
 
             if (iso_modes) {
-                int iso_mode_count = g_variant_n_children(iso_modes);
+                gsize iso_mode_count;
+                gint32 *modes = (int *)g_variant_get_fixed_array(iso_modes, &iso_mode_count, sizeof(gint32));
```

Comment:
Maybe use gint32 in the cast too, for consistency.

Reply:
Oops, I will fix that.